.github/actions/setup/action.yml

```diff
@@ -16,6 +16,9 @@ inputs:
   rustfmt:
     description: Install Rustfmt if `true`. Defaults to `false`.
     required: false
+  rust-toolchain:
+    description: Install Rust toolchain if `true`. Defaults to `false`.
+    required: false
```

Comment:
What I dislike about this input name is that it implies you're not installing any Rust toolchain without it. However, you'll get a Rust toolchain with the `clippy` or `rustfmt` options.

Reply:
What about this?
```
toolchain:
    description: Rust toolchain to install. Comma-separated string of [`build`, `format`, `lint`].
    required: false
```